pkg/webui/account/store/reducers/user.js

```diff
@@ -12,41 +12,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { INITIALIZE_SUCCESS } from '@ttn-lw/lib/store/actions/init'
+
 import {
   GET_USER,
   GET_USER_SUCCESS,
   GET_USER_FAILURE,
   UPDATE_USER_SUCCESS,
-  LOGOUT_FAILURE,
 } from '@account/store/actions/user'
 
 const defaultState = {
-  fetching: false,
   user: undefined,
-  error: false,
+  session_id: undefined,
```

Comment:
```suggestion
  sessionId: undefined,
```

Use camelcase unless forced otherwise.